<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyboardTrainer</class>
 <widget class="QMainWindow" name="KeyboardTrainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Клавиатурный тренажёр</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ChooseMode">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Выберите режим тренировки:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ModeDescription">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>450</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>описание</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ProgrammName">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>450</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Клавиатурный тренажёр.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Description">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>450</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Описание выбранного режима:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartTrainButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать тренировку!</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeTryAmount">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>165</width>
      <height>60</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Изменить кол-во попыток</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>311</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="LettersModeButton">
       <property name="text">
        <string>По буквам</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">ModeButtonsGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="WordsModeButton">
       <property name="text">
        <string>По словам</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">ModeButtonsGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ImageModeButton">
       <property name="text">
        <string>По предложениям</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">ModeButtonsGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>30</y>
      <width>20</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="DifficultyChangeBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>165</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Настроить % сложности</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowRecordsBtn">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>290</y>
      <width>165</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Рекорды</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeLettersCountBtn">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>350</y>
      <width>165</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Настроить количество букв</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="ModeButtonsGroup"/>
 </buttongroups>
</ui>
